WalletWasabi.Gui/Global.cs

```diff
@@ -82,6 +82,8 @@ public class Global
 
 		public Global(string dataDir, string torLogsFile, Config config, UiConfig uiConfig, WalletManager walletManager)
 		{
+			Guard.NotNull(nameof(config.Network), config.Network);
```

Comment:
Why does `Global` validates `config.Network`? Given that `Network` is a property of `Config` and can be assigned only by `Config` then it is `Config` the one that has to make sure its internal state is correct.

Why are we validating `config.Network` assuming that `config` is not null and the rest is not null neither? I thing we should validate in arguments as usual.

Reply:
Fixed